helm/hello-world-service/templates/backend-pvc.yaml

```diff
@@ -0,0 +1,34 @@
+{{- if and .Values.persistence.enabled (not .Values.persistence.existingClaim) }}
+{{- if .Values.persistence.efs.enabled -}}
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: "{{ .Release.Namespace }}-backend-volume"
+spec:
+  capacity:
+    storage: {{ .Values.persistence.baseSize | quote }}
+  accessModes:
```

Comment:
Please add the reclaim policy here also.

persistentVolumeReclaimPolicy: {{ .Values.persistence.reclaimPolicy | default "Recycle" }}

Reply:
Done. Thank you for the review!